chore(changelog): new way to maintain the changelog (#11279)
Kong/kong: .github/PULL_REQUEST_TEMPLATE.md

### Checklist

- [ ] The Pull Request has tests

> - [ ] Add a changelog file under the folder `CHANGELOG/unreleased/kong` if necessary. https://github.com/Kong/kong/CHANGELOG/README.md

Reviewer: ```suggestion
- [ ] A changelog file has been added to `CHANGELOG/unreleased/kong`. https://github.com/Kong/kong/CHANGELOG/README.md
```

> - [ ] A changelog file has been added to `CHANGELOG/unreleased/kong` or adding `skip-changelog` label on PR if unnecessary. [CHANGELOG/README.md](https://github.com/Kong/kong/CHANGELOG/README.md)

Reviewer: ```suggestion
- [ ] A changelog file has been added to `CHANGELOG/unreleased/kong` (You can ask @vm-001 to do it for you)
   /or/
- [ ] The `skip-changelog` label  has been added because no changelog entry is required
```
